generator/internal/codegen_utils_test.cc

```diff
@@ -196,28 +196,26 @@ TEST(ProcessCommandLineArgs, EmptyCommitHash) {
       "--cpp_codegen_opt=googleapis_commit_hash=<hash> must be specified.");
 }
 
-TEST(ProcessCommandLineArgs, NoCopyrighYearParameterOrValue) {
+TEST(ProcessCommandLineArgs, NoCopyrightYearParameterOrValue) {
   auto result = ProcessCommandLineArgs(
       "product_path=google/cloud/pubsub/,googleapis_commit_hash=foo");
-  auto expected_year =
-      absl::FormatTime("%Y", absl::Now(), absl::LocalTimeZone());
+  auto expected_year = CurrentCopyrightYear();
```

Comment:
It seems a little circular, but without injecting something I'm not sure what else to do.